### PR TITLE
[MRG] Add docstring to vtki backend

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
#### What does this implement/fix?
This PR updates the docstring/license/authors of the vtki backend and fix parameters.

### CODE REVIEW

File: mne/viz/backends/_pysurfer_mayavi.py [hunk 1]
```diff
@@ -43,7 +45,7 @@ def visible(self, state):
             self.pts.visible = state
 
 
-class _Renderer(object):
+class _Renderer(_BaseRenderer):
```
Comment (larsoner) [inline mne/viz/backends/_pysurfer_mayavi.py]: Now that you have put all of these in... I wonder if we should make a `copy_base_doc_to_subclass_doc` decorator instead. It iterates over public methods of the base class and populates the doc of public methods of the subclasses. WDYT?

Comment (GuillaumeFavelier) [reply mne/viz/backends/_pysurfer_mayavi.py]: That's the typical use-case for this class, I'll try to add this.

Comment (larsoner) [reply mne/viz/backends/_pysurfer_mayavi.py]: Looks much cleaner!

**Comment by larsoner** [comment]
Thanks @GuillaumeFavelier !

**Pull request merged**